### PR TITLE
feat: מצב היברידי — Cloud API (Arm A) + WPPConnect (Arm B)

### PR DESCRIPTION
**Pull request by amirbiron**
## תיאור קצר
הוספת תמיכה במצב היברידי של WhatsApp: Cloud API (Meta) לשליחת הודעות פרטיות (Arm A) ו-WPPConnect לשליחת הודעות לקבוצות (Arm B). כולל webhook handler חדש, PyWaProvider, ניתוב חכם לפי סוג יעד, וקישורי wa.me לתפיסת משלוחים.

## שינויים עיקריים
- [x] קוד (Backend / Services)
- [x] בוט וואטסאפ
- [x] Celery / Workers

**פירוט:**
- **webhook handler חדש** (`app/api/webhooks/whatsapp_cloud.py`): קבלת הודעות מ-Meta Cloud API, אימות חתימת HMAC-SHA256, חילוץ טקסט ומדיה, idempotency, ניתוב ל-state machine handlers
- **PyWaProvider** (`app/domain/services/whatsapp/pywa_provider.py`): מימוש ממשק BaseWhatsAppProvider מעל Cloud API, תמיכה ב-inline buttons, מדיה עשירה, retry עם circuit breaker
- **Provider Factory** (`app/domain/services/whatsapp/provider_factory.py`): ניתוב חכם — Cloud API לפרטי, WPPConnect לקבוצות, circuit breaker נפרד לכל ספק
- **wa.me Link Generation** (`app/domain/services/whatsapp/wa_me_links.py`): יצירת קישורי Click-to-Chat לתפיסת משלוחים דרך Cloud API
- **ניתוב היברידי** (`app/api/webhooks/whatsapp.py`, `app/workers/tasks.py`): זיהוי קבוצות (@g.us) וניתוב לספק המתאים
- **תצורה** (`app/core/config.py`): הוספת משתנים לـ Cloud API (token, phone ID, verify token, app secret)
- **Circuit Breaker** (`app/core/circuit_breaker.py`): circuit breaker נפרד לـ Cloud API
- **Outbox Service** (`app/domain/services/outbox_service.py`): הוספת קישור wa.me לשידורי משלוחים

## בדיקות
- [x] Unit tests — עוברים
  - `tests/test_pywa_provider.py`: 330 שורות — בדיקות שליחת טקסט, מדיה, retry, keyboard fallback, normalization
  - `tests/test_whatsapp_cloud_webhook.py`: 189 שורות — אימות webhook, חתימת HMAC, חילוץ טקסט/מדיה
  - `tests/test_hybrid_routing.py`: 153 שורות — זיהוי קבוצות, ניתוב לספק הנכון
- [x] כיסוי ל-edge cases: טקסט ריק, כפתורים יותר מ-3, חתימה שגויה, מדיה חסרה

## CI Checks
- pytest — כל הבדיקות עוברות
- ולידציית דיאגרמות (אם קיימות)

## סוג שינוי
- [x] feat: פיצ'ר חדש

## צ'קליסט
- [x] כל הפלט בעברית (כ

https://claude.ai/code/session_0114yN7wb17ePJNiTegdCsfu

### CODE REVIEW

File: app/api/webhooks/whatsapp_cloud.py [hunk 1]
```diff
@@ -0,0 +1,599 @@
...
+            response, new_state = await _route_to_role_menu_wa(user, db, state_manager)
+        background_tasks.add_task(
+            send_whatsapp_message, reply_to, response.text, response.keyboard
+        )
+        return response.text, new_state
+
+    # ניתוב לתפריט סדרן (כפתור "תפריט סדרן" — פתוח לכל מי שהוא סדרן פעיל)
+    # בדיקת keyword רק כשהמשתמש לא באמצע זרימת סדרן — מונע תפיסת טקסט חופשי
+    _in_dispatcher_flow = isinstance(current_state, str) and current_state.startswith("DISPATCHER.")
+    if text and not _in_dispatcher_flow and ("תפריט סדרן" in text or "🏪 תפריט סדרן" in text):
```
[the remark this answers is not in the document]
Comment (amirbiron) [reply app/api/webhooks/whatsapp_cloud.py]: לא קריטי
